### ISSUE TITLE
Add proper support for Android N split screen

### ISSUE THREAD

**Issue by mvysny** [mvysny/aedict#738]
On Android N, the warning of "this app may not work with split-screen" appears. Aedict does seem to function completely if I'm already in progress with a search, BUT it seems to do nothing if I perform a search from the initial screen upon starting the app. Nor does it let me scroll in this case.  I'm not sure how much effort it takes to update an app to compatibility with a brand new Android function, but your app seems to MOSTLY be there already. Please look into this! Thanks a ton.

**Comment by mvysny** [comment]
Blocked by #740

**Comment by mvysny** [comment]
Quoting Android documentation at https://developer.android.com/guide/topics/ui/multi-window.html

> If you build a multi-orientation app that targets API level 23 or lower, and the user uses the app in multi-window mode, the system forcibly resizes the app. The system presents a dialog box warning the user that the app may behave unexpectedly.

That's too bad. In order to properly support Multi-Window/Split-Screen, I need to build for API 24 and higher. However that activates those damn broken lazy permissions which are not easy to support properly. I need to reevaluate that.

**Comment by mvysny** [comment]
Blocked by #741

**Comment by mvysny** [comment]
Won't fix because of #741
